src/hooks/usePopper.ts

```diff
@@ -0,0 +1,136 @@
+import { useRef, useEffect, RefObject, useState, useMemo, useLayoutEffect } from 'react'
+import { Instance, Options, State, Placement } from '@popperjs/core'
+
+export const TOOLTIP_OFFSET = 8 // px
+
+const defaultConfig: Partial<Options> & Pick<Options, 'modifiers'> = {
+  // default tooltip placement
+  placement: 'top',
+  modifiers: [
+    {
+      // move slightly to make space for the arrow element
+      name: 'offset',
+      options: {
+        offset: [0, 8],
```

Comment:
Is this 8 supposed to be the constant above?

Reply:
Forgot to inckude, it's `TOOLTIP_OFFSET`